.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+# Pull Request Document
+
+Closes # (issue)
+
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+## Type of change
+
+What type of change is this PR is related to? (select one or more)
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Steps to reproduce
+
+Please explain what needs to be tested in order to verify your changes.
+
+1. Do this and then do that
+2. Finally, do this
+
+## Checklist
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings (If do please explain)
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
```

Comment:
add 
```
[ ] Jenkins build passed
[ ] Tested solution on staging vm
```

So we can make sure every step needed was done [later, we should be able to connect pipelines and this would be removed]

Reply:
Ok, looks nice